bodhi/server/models.py

```diff
@@ -827,7 +827,7 @@ def mandatory_days_in_testing(self):
             if days:
                 return days
         days = int(config.get('%s.mandatory_days_in_testing' %
-                              self.id_prefix.lower().replace('-', '_')))
+                              self.id_prefix.lower().replace('-', '_'), 0))
         if not days:
```

Comment:
Now we never get into this case.
Maybe instead, do:
````
days = config.get('....')
if not days:
    log.warn(....)
    return 0
else:
    return int(days)
````

Reply:
Actually, how about `if days it not None:`, since you might have days=0.

Reply:
I have made it so.